### PR TITLE
Move SDK version to 4.3.2 for testing support.

### PR DESCRIPTION
**Pull request by serkantkaraca**
Move SDK version to 4.3.2 for testing support.

### CODE REVIEW

**Review by jsquire — APPROVED** [review]
LGTM, assuming that you don't want to use the default "make this a beta in case we accidentally release" scheme.  

Going to close out the automated version of this (#16433) since it's not needed.

**Comment by serkantkaraca** [comment]
> LGTM, assuming that you don't want to use the default "make this a beta in case we accidentally release" scheme.
> 
> Going to close out the automated version of this (#16433) since it's not needed.

Thanks for pointing. Actually second opinion, better to take the automated version. I will reopen it.

**Pull request closed without merging**
